### PR TITLE
Adding PHPFUI/ORM benchmark

### PR DESCRIPTION
**Pull request by phpfui**
Interesting benchmark suite.  Most of these benchmark suites don't work due to bit rot.  I had no issues once I set $_ENV correctly, as I don't have Docker installed on my machine.

I added PHPFUI/ORM.

On my Windows machine, I am getting the following times:
PHPFUI/ORM 10000 CRUD operations
Time: 7.09 seconds
Memory: 172.97 Kilobytes
Memory Peak: 2076.2 Kilobytes

Times and memory usage for the other benchmarks you list were very close to what I reproduced on my machine, so the above numbers should be similar to what you see.

### CODE REVIEW

**Comment by compositephp** [comment]
Hi,
What do you mean by "bit rot"? All tests are still executing normally.
For your ORM, I have the following results on the same hardware used for the previous tests:
```
PHPFUI/ORM 10000 CRUD operations
Time: 20.43 seconds
Memory: 109.37 Kilobytes
Memory Peak: 2196.59 Kilobytes
```
I would be happy to merge your pull request. Please have a look at the comment I left on your diff.

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "name": "compositephp/performance",
     "autoload": {
         "psr-4": {
-            "App\\": "src/"
+            "App\\": "src/",
+            "PHPFUIORM\\": "src/PHPFUIORM/"
```
Comment (compositephp) [inline composer.json]: Is it possible to avoid new namespace definition?

Comment (phpfui) [reply composer.json]: Yes, fixed in latest commit.  I did not realize you were using more of a library type style for the benchmarks.

**Comment by phpfui** [comment]
Sorry, I was referring to other benchmarks suites with the "bit rot".  Yours is one of the few that has not rotted!

Thanks for adding this.

**Pull request merged**